samples/kotlin-webmvc/build.gradle

```diff
@@ -1,10 +1,10 @@
 buildscript {
     ext {
-        kotlinVersion = "1.2.60"
+        kotlinVersion = "1.3.0"
         springAutoRestDocsVersion = "2.0.3-SNAPSHOT"
         springRestDocsVersion = "2.0.2.RELEASE"
         springBootVersion = "2.0.6.RELEASE"
-        dokkaVersion = "0.9.16"
+        dokkaVersion = "0.9.17"
```

Comment:
@fbenz FYI if downstream project uses `< 0.9.17`, build fails on 
```
Execution default of goal org.jetbrains.dokka:dokka-maven-plugin:0.9.16:dokka failed: An API incompatibility was encountered while executing org.jetbrains.dokka:dokka-maven-plugin:0.9.16:dokka: java.lang.NoSuchMethodError: org.jetbrains.dokka.DocumentationOptions.<init>(Ljava/lang/String;Ljava/lang/String;ZZZZZIZLjava/util/List;Ljava/util/List;Ljava/util/List;Ljava/util/List;ZLjava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/List;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
```
everyone who now uses 2.0.3-SNAPSHOT will get this unexpected error.

Reply:
Yes, this is expected. You have to use the same Dokka version as our extension. Dokka is built that way.